dev/fattest.simplicity/bnd.bnd

```diff
@@ -11,6 +11,9 @@
 -include= ~../cnf/resources/bnd/bundle.props
 bVersion=1.0
 
+javac.source: 1.8
```

Comment:
we can't upgrade the test framework to Java 8 quite yet, since some builds still need to run on Java 7